build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'maven'
 apply plugin: 'signing'
 
 group = 'com.cloudant'
-version = '1.2.0'
+version = '1.2.1-SNAPSHOT'
```

Comment:
Does this mean that somebody forgot to bump the version when the last release was done?